bundled-guava/src/main/java/org/apache/iceberg/GuavaClasses.java

```diff
@@ -81,6 +82,7 @@
     Multimaps.class.getName();
     Ordering.class.getName();
     Sets.class.getName();
+    Stopwatch.class.getName();
```

Comment:
We are trying to avoid adding more Guava classes. Could you use `System.currentTimeMilis()` instead?

Reply:
Done